rtv/docs.py

```diff
@@ -43,3 +43,17 @@
   `LEFT` or `h`       : Return to subreddit mode
   `RIGHT` or `l`      : Fold the selected comment, or load additional comments
 """
+
+COMMENT_FILE = """
+--% {0} %--
+Do not edit the line above.
+Type your comment above the line.
+All text below the line will be discarded.
+
+If there is nothing but whitespace above the line
+then the comment will be discarded.
+
+Replying to {1}'s {2}
+
+{3}
+"""
```

Comment:
That's an interesting point. I think the timing isn't as important here because this isn't performance critical code and we're talking about a difference between 5 us and 0.5 us. My comment was more concerned about code readability. It's a lot easier for somebody skimming through to understand the context surrounding ``--% $uuid %--`` compared to ``--% {0} %--``. Or if you prefer the standard string formatting you can do ``--% {uuid} %--``, either way works for me.

BTW what's the purpose of including a uuid? It doesn't appear to be linked to anything relevant. Just brainstorming, here's what I came up with copying Git's template.

```
# Please enter a comment. Lines starting with '#' will be ignored,
# and an empty message aborts the commit.
#
# Replying to {message_type} by {author}:
#         {text}
```
Are you familiar with any other command line programs that use this technique? I looked at Mercurial and it just looks like an uglier version of Git's message.